app.json

```diff
@@ -27,7 +27,7 @@
       "supportsTablet": true
 		},
 		"extra": {
-			"variant": "donor"
+			"variant": "client"
```

Comment:
ideally, you exclude changes like these from your commits

Reply:
Reverted this to original.